packages/hardhat-core/src/internal/hardhat-network/provider/types/Block.ts

```diff
@@ -10,11 +10,12 @@ export const Block: BlockConstructor = EthBlock;
 
 // tslint:disable-next-line:no-misused-new
 type BlockConstructor = new (
-  data?: BlockData | null,
+  data?: BlockData | null | [Buffer[], Buffer[], Buffer[]],
   chainOptions?: { common: Common }
 ) => Block;
 
 export interface Block {
+  readonly raw: [Buffer[], Buffer[], Buffer[]];
```

Comment:
I don't understand the changes in this file.

Reply:
I copy-pasted the missing bits from the upstream definition of Block. I defined it upstream IIRC.

Reply:
Oh, this isn't used in this PR anymore!

Reply:
I reverted the change.